index.html

```diff
@@ -99,19 +89,17 @@ <h2>
         <ul class="card-work-row1">
           <h1 class="primary-text">My Recent Works</h1>
           <article class="card">
-              <div class="card-image">
-                <img class="img-plaholders" src="assets/images/Img-plaholder.svg" alt="Img-plaholder" />
-              </div>
-              <h2 class="project-title">Multi-Post Stories Gain+Glory</h2>
-              <ul class="tags">
-                <li class="tag">HTML</li>
-                <li class="tag">JS</li>
-                <li class="tag">CSS</li>
-                <li class="tag">Ruby & Rails</li>
-              </ul>
-              <button class="btn-popup" type="button" title="see-project">
-                See Project
-              </button>
+            <div class="card-image">
+              <img class="img-plaholders" src="assets/images/Img-plaholder.svg" alt="Img-plaholder" />
+            </div>
+            <h2 class="project-title">Multi-Post Stories Gain+Glory</h2>
+            <ul class="tags">
+              <li class="tag">HTML</li>
+              <li class="tag">JS</li>
+              <li class="tag">CSS</li>
+              <li class="tag">Ruby & Rails</li>
+            </ul>
+            <button class="btn-popup" type="button" title="see-project">See Project</button>
```

Comment:
- [ ] Please create your projects section dynamically using JavaScript, as mentioned in the project's requirements:

![image](https://github.com/yusufyusie/yusufyusie.github.io/assets/105105262/2bd41b94-c8b3-42e1-9b17-e915a8224f74)

It means that, on your main page, you need to populate all of your project cards dynamically using JavaScript. For this, you need to use the data that you have stored in the array of objects. You need to remove the relevant HTML and display the projects with JavaScript by using the data that you have stored in the array of objects. The same data related to the project will be shown in the pop-up window when that one is clicked. By doing so, adding projects in the future will be easy. You will only need to add it in one place - as another object in the projects array. For help, please have a look at [this](https://github.com/olawale-o/olawale-o.me/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95) PR.